src/test/java/enmasse/kafka/bridge/example/BridgeReceiver.java

```diff
@@ -139,7 +139,8 @@ public void run(Vertx vertx) {
 								Object partition = messageAnnotations.getValue().get(Symbol.getSymbol("x-opt-bridge.partition"));
 								Object offset = messageAnnotations.getValue().get(Symbol.getSymbol("x-opt-bridge.offset"));
 								Object key = messageAnnotations.getValue().get(Symbol.getSymbol("x-opt-bridge.key"));
-								LOG.info("... on partition {} [{}], key = {}", partition, offset, key);
+								Object topicName = messageAnnotations.getValue().get(Symbol.valueOf("x-opt-bridge.topic"));
```

Comment:
can you change this just to "topic" (not "topicName") to be consistent with the annotation that is ok.